go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/prometheus/client_golang v1.15.0
 	github.com/riferrei/srclient v0.5.4
 	github.com/shopspring/decimal v1.3.1
-	github.com/smartcontractkit/libocr v0.0.0-20230802221916-2271752fa829
+	github.com/smartcontractkit/libocr v0.0.0-20230922131214-122accb19ea6
```

Comment:
FWIW Looks like head is `ffa38fe11ef8e9571e20f4ce4166e7010cfbe885` but insignificant difference :shrug: